rules/docker-sonic-mgmt-framework.mk

```diff
@@ -0,0 +1,34 @@
+# docker image for mgmt-framework
+
+DOCKER_MGMT_FRAMEWORK_STEM = docker-sonic-mgmt-framework
+DOCKER_MGMT_FRAMEWORK = $(DOCKER_MGMT_FRAMEWORK_STEM).gz
+DOCKER_MGMT_FRAMEWORK_DBG = $(DOCKER_MGMT_FRAMEWORK_STEM)-$(DBG_IMAGE_MARK).gz
+
+$(DOCKER_MGMT_FRAMEWORK)_PATH = $(DOCKERS_PATH)/$(DOCKER_MGMT_FRAMEWORK_STEM)
+
+$(DOCKER_MGMT_FRAMEWORK)_DEPENDS += $(REDIS_TOOLS) $(SONIC_MGMT_FRAMEWORK)
+$(DOCKER_MGMT_FRAMEWORK)_DBG_DEPENDS = $($(DOCKER_CONFIG_ENGINE_STRETCH)_DBG_DEPENDS)
+$(DOCKER_MGMT_FRAMEWORK)_DBG_DEPENDS += $(REDIS_TOOLS) $(SONIC_MGMT_FRAMEWORK_DBG)
+
+SONIC_DOCKER_IMAGES += $(DOCKER_MGMT_FRAMEWORK)
+$(DOCKER_MGMT_FRAMEWORK)_LOAD_DOCKERS += $(DOCKER_CONFIG_ENGINE_STRETCH)
+$(DOCKER_MGMT_FRAMEWORK)_DBG_IMAGE_PACKAGES = $($(DOCKER_CONFIG_ENGINE_STRETCH)_DBG_IMAGE_PACKAGES)
+
+ifeq ($(ENABLE_MGMT_FRAMEWORK), y)
+SONIC_INSTALL_DOCKER_IMAGES += $(DOCKER_MGMT_FRAMEWORK)
+SONIC_STRETCH_DOCKERS += $(DOCKER_MGMT_FRAMEWORK)
+endif
+
+SONIC_DOCKER_DBG_IMAGES += $(DOCKER_MGMT_FRAMEWORK_DBG)
+ifeq ($(ENABLE_MGMT_FRAMEWORK), y)
+SONIC_INSTALL_DOCKER_DBG_IMAGES += $(DOCKER_MGMT_FRAMEWORK_DBG)
+SONIC_STRETCH_DBG_DOCKERS += $(DOCKER_MGMT_FRAMEWORK_DBG)
+endif
+
+$(DOCKER_MGMT_FRAMEWORK)_CONTAINER_NAME = mgmt-framework
+$(DOCKER_MGMT_FRAMEWORK)_RUN_OPT += --net=host --privileged -t
+$(DOCKER_MGMT_FRAMEWORK)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
+$(DOCKER_MGMT_FRAMEWORK)_RUN_OPT += -v /etc:/host_etc:ro
+$(DOCKER_MGMT_FRAMEWORK)_RUN_OPT += --mount type=bind,source="/var/platform/",target="/mnt/platform/"
```

Comment:
why do we need to mount platform in mgmt framework. it suppose to be platform independent, right?